lib/cryptosuites/suites.go

```diff
@@ -223,7 +224,7 @@ var (
 		ProxyToDatabaseAgent:    ECDSAP256,
 		ProxyKubeClient:         ECDSAP256,
 		EC2InstanceConnect:      Ed25519,
-		GitHubProxyCASSH:        Ed25519,
+		GitHubProxyCASSH:        ECDSAP256,
```

Comment:
I'd stick with Ed25519 here, we explicitly don't support the balanced-v1 suite and will return an error if using HSM or KMS with balanced-v1 suite https://github.com/gravitational/teleport/blob/76ac553af501500322a5a7c0e1f2ae437aeea577/api/types/authentication.go#L661-L666